docs/contributing/05-gh-actions.md

```diff
@@ -2,205 +2,118 @@
-
-- openSSF Scorecard rating (16.08.2023)
-  - [ahmadnassri/action-workflow-queue](./scorecard-ratings/ahmadnassri_action-workflow-queue.json): 6.5/10
-
-#### Versions
-
-- ahmadnassri/action-workflow-queue@v1.1.5
-- ahmadnassri/action-workflow-queue@0144da6a252d1986d817826d963ad3a9af1e7fdf
-
-### autotelic/action-wait-for-status-check
-
-- Repository: <https://github.com/autotelic/action-wait-for-status-check>
-- Marketplace: <https://github.com/marketplace/actions/wait-for-github-status-check>
-
-#### Rating
-
-- openSSF Scorecard rating (16.08.2023)
-  - [autotelic/action-wait-for-status-check](./scorecard-ratings/autotelic_action-wait-for-status-check.json): 4.8/10
-
-#### Versions
-
-- autotelic/action-wait-for-status-check@6556cf50c8fb6608412945382eae73581f56cbb4
-- autotelic/action-wait-for-status-check@v1
-
-### gaurav-nelson/github-action-markdown-link-check
-
-- Repository: <https://github.com/gaurav-nelson/github-action-markdown-link-check>
-- Marketplace: <https://github.com/marketplace/actions/markdown-link-check>
-- `markdown-link-check` repository: <https://github.com/tcort/markdown-link-check>
-
-The action runs a shell script which installs and runs a Node.js module called markdown-link-check. The [version of the Node.js module is tagged](https://github.com/gaurav-nelson/github-action-markdown-link-check/blob/46e442156b8161bfd0913357c7a411e0e610d2ad/entrypoint.sh#L11).
-
-#### Rating
-
-- openSSF Scorecard rating (16.08.2023)
-  - [gaurav-nelson/github-action-markdown-link-check](./scorecard-ratings/gaurav-nelson_github-action-markdown-link-check.json): 4.4/10
-  - [tcort/markdown-link-check](./scorecard-ratings/tcort_markdown-link-check.json): 4.6/10
-
-#### Versions
-
-- gaurav-nelson/github-action-markdown-link-check@v1
-- gaurav-nelson/github-action-markdown-link-check@d53a906aa6b22b8979d33bc86170567e619495ec
-
-### slackapi/slack-github-action
-
-- Repository: <https://github.com/slackapi/slack-github-action>
-- Marketplace: <https://github.com/marketplace/actions/slack-send>
+## GitHub Actions Security Best Practices
 
-#### Ratings
+Even though the usage of GitHub Actions is restricted, some threats remain. The responsibility to target these threats is on the developer using GitHub Actions.
 
-- openSSF Scorecard rating (16.08.2023)
-  - [slackapi/slack-github-action](./scorecard-ratings/slackapi_slack-github-action.json): 6.1 / 10
+### Security review
 
-#### Versions
+The code of the action has to be reviewed to identify suspicious parts of the code. It should be especially checked if the Action is processing secrets beside the expected usage and what kind of modification the Action does on Github resources.
 
-- slackapi/slack-github-action@e28cf165c92ffef168d23c5c9000cffc8a25e117 # v1.24.0,
+### Set the minimal scope for credentials
 
-## Not allowed GitHub Actions
+- GITHUB_TOKEN has read and write permissions
+- Must be restricted to the least minimum
+- Also valid for all other credentials used by the Github Action
```

Comment:
I don't think this section should be arranged into points. What does "Must be restricted to the least minimum" refer to?

Reply:
Created sentences out of the text arranged in points.